DirectEmployers/MyJobs
# Disable AuthorizeSauce's debug setting when in production [PD-3235]

## What's this PR do?
Disables the debug setting for credit card authorization when in production. Credit card transactions are being rejected because we're posting to authorize.net's sandbox api.
## Screenshots (if appropriate)

## Where should the reviewer start?

## How should this be manually tested?
The best we can do for now in staging is verify we haven't broken the sandbox api requests. We can create a payed product for apply.directemployers.org in the staging environment and test that we're able to purchase it from another company.
## Any background context you want to provide?
